<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradesWindow</class>
 <widget class="QMainWindow" name="UpgradesWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Upgrades</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="VegetablesBEUpgrade">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расширить склад
овощей
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FruitsBEUpgrade">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расширить склад
фруктов
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BerriesBEUpgrade">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расширить склад
ягод
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VegetablesUpgrade">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Улучшить склад
овощей
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FruitsUpgrade">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Улучшить склад
фруктов
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BerriesUpgrade">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Улучшить склад
овощей
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ContractUpgrade">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>192</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сменить контракт на
более выгодный
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
